### PR TITLE
SSTTP-1067 - Changed tests in calculator form to use new form updated

### PR DESCRIPTION
**Pull request by cavster**
Note we need to write a new ticket to remove the other form as logic is duplicated

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -118,12 +118,16 @@ ssttp.calculator.form.what-you-owe-amount.amount.required=Enter an amount of £0
 ssttp.calculator.form.what-you-owe-amount.amount.less-than-maxval=Total amounts must be less than £1,000,000,000,000
 
 ssttp.calculator.form.what-you-owe-date.title=What''s the due date of the amount you owe?
+ssttp.calculator.form.what-you-owe.due_by.not-valid-date=You need to enter a valid date
 ssttp.calculator.form.what-you-owe-date.due_by.not-valid-year-too-low=Year must be 1996 or later
 ssttp.calculator.form.what-you-owe-date.due_by.not-valid-year-too-high=Year is too far in the future
-ssttp.calculator.form.what-you-owe-date.due_by.not-valid-month=You need to enter a valid month
-ssttp.calculator.form.what-you-owe-date.due_by.not-valid-day=You need to enter a valid day
+ssttp.calculator.form.what-you-owe-date.due_by.not-valid-year=Enter a valid Year
```
Comment (Smudgey) [inline conf/messages]: year should be lowercase

File: conf/messages [hunk 2]
```diff
@@ -118,12 +118,16 @@ ssttp.calculator.form.what-you-owe-amount.amount.required=Enter an amount of £0
 ssttp.calculator.form.what-you-owe-amount.amount.less-than-maxval=Total amounts must be less than £1,000,000,000,000
 
 ssttp.calculator.form.what-you-owe-date.title=What''s the due date of the amount you owe?
+ssttp.calculator.form.what-you-owe.due_by.not-valid-date=You need to enter a valid date
```
Comment (Smudgey) [inline conf/messages:121]: Is this being used anymore?

**Review by Smudgey — CHANGES_REQUESTED** [review]
1 change and 1 question, otherwise all is fine

File: conf/messages (hunk 2 above)
Comment (cavster) [reply conf/messages:121]: Yes

**Review by Smudgey — APPROVED** [review]

**Pull request merged**